account_move_name_sequence/views/account_journal.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--
+  Copyright 2021 Akretion France (http://www.akretion.com/)
+  @author: Alexis de Lattre <alexis.delattre@akretion.com>
+  License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+-->
+<odoo>
+
+    <record id="view_account_journal_form" model="ir.ui.view">
+        <field name="model">account.journal</field>
+        <field name="inherit_id" ref="account.view_account_journal_form" />
+        <field name="arch" type="xml">
+            <field name="refund_sequence" position="before">
+                <field
+                    name="sequence_id"
+                    required="0"
+                    context="{'default_name': name, 'default_company_id': company_id, 'default_implementation': 'no_gap', 'default_padding': 4, 'default_use_date_range': True, 'default_prefix': code + '/%%(range_year)s/'}"
```

Comment:
If you click on `sequence_id` before setting up `code`, throws an error.

I suggest:
1. Fallback with a default text: `'default_prefix': (code or 'UNKNOWN') + '/%%(range_year)s/'`
2. Invisible by attrs: `attrs="{'invisible': [('code', '=', False)]}"`